File: fizzbuzz/fizzbuzz.swift
func fizzbuzz(n: Int) {
  for i in 1...n {
    if i % 3 == 0 && i % 5 == 0 {
      print("fizzbuzz")
    } else if i % 3 == 0 {
      print("fizz")
    } else if i % 5 == 0 {
      print("buzz")
    } else {
      print(i)
    }
  }
}

// Entry point
func main() {
  fizzbuzz(n: 100)
}

// Call the entry point
main()

// how to run:
// swiftc -o fizzbuzz-swift fizzbuzz.swift ; ./fizzbuzz-swift

File: fizzbuzz/fizzbuzz.rs
/*! 
  FizzBuzz program by Danya&Dima, using Google Bard
  not yet yested
*/

fn fizzbuzz(n: u32) {
  for i in 1..n + 1 {
    let output = match i {
      x if x % 15 == 0 => "FizzBuzz",
      x if x % 3 == 0 => "Fizz",
      x if x % 5 == 0 => "Buzz",
      _ => i,
    };
 
    println!("{}", output);
  }
}
 
fn main() {
  fizzbuzz(100);
}
